CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## v2.11.0 / 2024-02-19
+
+### Note
+
+This release builds with Golang v1.21.7.
+
+ef8f1bfa: Dale Henries <dhenries@samaritan.org>: docs: add OpenSSF Scorecard to README
```

Comment:
Why did you change the format of the changelog here? Feel free to reuse https://github.com/kubernetes/kube-state-metrics/pull/2259

Reply:
Ah, I wanted to adhere to a format that can be generated by `git`, as that would be essential in [automating releases](https://github.com/kubernetes/kube-state-metrics/pull/2325/files#diff-226a12cd45d08a9b54de8d6c5bdfd74c461520624b2e2174806ef73d954f6c10R170).

Reply:
I would prefer, if we discuss these changes after the release and keep the current release process for now

Reply:
I've made the changes here and updated [#2325](https://github.com/kubernetes/kube-state-metrics/pull/2325/files#r1510879597) as well.

Reply:
Thanks!